src/frontend/packages/lib_video/src/api/pollForLive/index.spec.tsx

```diff
@@ -11,7 +11,6 @@ describe('createPlayer', () => {
   });
 
   afterEach(() => {
-    jest.runOnlyPendingTimers();
```

Comment:
A changed the commit message:

> ✅(lib-video) fix missing api mock on pollForLive test
> 
> A mock to https://marsha.education/live.m3u8 was triggered in
> the pollForLive test (Unmatched GET to https://marsha.education/live.m3u8)
> but not capture by fetchMock because of "jest.runOnlyPendingTimers();"
> This commit fixes that.

Reply:
This commit fixes the error 500: https://github.com/openfun/marsha/pull/2135/commits/ab53a1553bfc34b2d7d7ed8dde7002d511a17729